packages/dashboard/test/dashboard.test.ts

```diff
@@ -169,6 +169,90 @@ describe('dashboard', () => {
     });
   });
 
+  describe('i18n', () => {
+    it('should have default values', () => {
+      expect(dashboard.i18n).to.eql({
+        selectSectionTitleForEditing: 'Select section title for editing',
+        selectWidgetTitleForEditing: 'Select widget title for editing',
+        remove: 'Remove',
+        resize: 'Resize',
+        resizeApply: 'Apply',
+        resizeShrinkWidth: 'Shrink width',
+        resizeGrowWidth: 'Grow width',
+        resizeShrinkHeight: 'Shrink height',
+        resizeGrowHeight: 'Grow height',
+        move: 'Move',
+        moveApply: 'Apply',
+        moveForward: 'Move Forward',
+        moveBackward: 'Move Backward',
+      });
+    });
+
+    it('should localize widget', async () => {
+      dashboard.i18n = {
+        ...dashboard.i18n,
+        selectWidgetTitleForEditing: 'foo',
+      };
+
+      await nextFrame();
+
+      const widget = getElementFromCell(dashboard, 0, 0) as DashboardWidget;
+      expect(widget.i18n.selectWidgetTitleForEditing).to.equal('foo');
+      expect(widget.i18n).to.eql({
```

Comment:
Nit: Can be simplified to `expect(section.i18n).to.eql(dashboard.i18n);`

Reply:
Done